src/components/Single-review.jsx

```diff
@@ -1,25 +1,33 @@
 import { useEffect, useState } from 'react';
 import { useParams } from 'react-router-dom';
-import { getSingleReview } from './api';
+import { getSingleReview, getCommentById } from './api';
 
 export const SingleReview = () => {
 	const [review, setReview] = useState([]);
+	const [comments, setComments] = useState([]);
 	let { review_id } = useParams();
 	const [isLoading, setIsLoading] = useState(true);
 	const [error, setError] = useState(null);
+	const [isComments, setIsComments] = useState(false);
+
+	const handleOnClick = () => {
+		setIsComments((isComments) => !isComments);
+	};
 
 	useEffect(() => {
 		const fetchData = async () => {
 			try {
 				const data = await getSingleReview(review_id);
+				const commentData = await getCommentById(review_id);
```

Comment:
As you've now put these fetch requests together, be aware that one failing means that the whole page will display the error message to the user. Instead, you may want to display just the review if getSingleReview resolves without an error but getCommentById fails. It would mean tweaking your error states, but it would provide a better user experience in the long run